pyproject.toml

```diff
@@ -21,6 +22,13 @@ mypy = { version='*', optional=true}
 pytest = { version='*', optional=true }
 pytest-cov = { version='*', optional=true }
 types-PyYAML = { version='*', optional=true }
+# optional documentation dependencies
+sphinx = { version='*', optional=true}
+sphinx-rtd-theme = { version='*', optional=true}
+bluepy = { version='*', optional=true}
```

Comment:
remove bluepy dependency; not required and causes issues with local install

Reply:
fixed